docker/bookkeeper/entrypoint.sh

```diff
@@ -9,6 +9,8 @@
 #     http://www.apache.org/licenses/LICENSE-2.0
 #
 
+set -eo pipefail
```

Comment:
+1

Reply:
These flags try to make the script safer:

- `-e` option causes a bash script to exit immediately when a command fails.
- `-o pipefail` option sets the exit code of a pipeline to that of the rightmost command to exit with a non-zero status. Or to zero if all commands of the pipeline exit successfully.